### PR TITLE
Finnish fixes 20

### PR DESCRIPTION
**Pull request by fergusq**

### CODE REVIEW

File: mem-01-b.xml [hunk 1]
```diff
@@ -3466,18 +3466,22 @@ Sanoa "kun olin X-vuotias", rakennus on esimerkiksi {qaStaHvIS DISwIj javDIch...
...
-      <column name="notes_fa">این به عضوی از شرکت کشتی ({wey:n}) اشاره دارد که افسر نیست ({yaS:n}).[2] [AUTOTRANSLATED]</column>
-      <column name="notes_sv">Detta avser en medlem i ett fartygs företag ({wey:n}) som inte är officer ({yaS:n}).[2] [AUTOTRANSLATED]</column>
-      <column name="notes_ru">Это относится к члену судовой компании ({wey:n}), который не является офицером ({yaS:n}).[2] [AUTOTRANSLATED]</column>
-      <column name="notes_zh_HK">這是指不是高級船員（{yaS:n}）的船舶公司成員（{wey:n}）。[2] [AUTOTRANSLATED]</column>
-      <column name="notes_pt">Refere-se a um membro de uma companhia marítima ({wey:n}) que não é um oficial ({yaS:n}) .[2] [AUTOTRANSLATED]</column>
-      <column name="notes_fi">Tämä viittaa aluksen miehistön ({wey:n}) jäseneen, joka ei ole upseeri ({yaS:n}).[2]</column>
+      <column name="notes">This refers to a member of a ship's company ({wey:n}) who is not an officer ({yaS:n}).[2]
+
+This can also refer to a member of a loosely organized group with no governmental- or military-like status.[3]</column>
```
[the remark this answers is not in the document]
Comment (fergusq) [reply mem-01-b.xml:3475]: http://klingon.wiki/Word/Ku-vIgh

File: mem-06-j.xml [hunk 2]
```diff
@@ -3767,18 +3767,18 @@ Huomaa, että {jemS:n:nolink} ja {qIrq:n:nolink} eivät noudata oikeaa klingonif
       <column name="definition_ru">Отжимания</column>
       <column name="definition_zh_HK">俯卧撑 [AUTOTRANSLATED]</column>
       <column name="definition_pt">Flexão</column>
-      <column name="definition_fi">Etunojapunnerrus</column>
+      <column name="definition_fi">punnerrus</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{ngolpel:n}</column>
-      <column name="notes"></column>
+      <column name="notes">This word can be the object of {mI':v}.</column>
```
[the remark this answers is not in the document]
Comment (fergusq) [reply mem-06-j.xml:3774]: http://klingon.wiki/Word/Ngolpel

File: mem-01-b.xml [hunk 3]
```diff
@@ -3466,18 +3466,22 @@ Sanoa "kun olin X-vuotias", rakennus on esimerkiksi {qaStaHvIS DISwIj javDIch...
...
+      <column name="notes_de"></column>
```
[the remark this answers is not in the document]
Comment (fergusq) [reply mem-01-b.xml:3476]: I tried to run autotranslation, but for some reason it crashes. I don't remember the exact error right now, but it had to do with a regex not matching and it came from inside the googletrans package.

[the remark this answers is not in the document]
Comment (De7vID) [reply mem-01-b.xml:3476]: Yeah, the package isn't "official" and Google Translate API changes sometimes break it.

File: mem-01-b.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (De7vID) [reply mem-01-b.xml:3475]: Ah, some of those links weren't there when I saw the original new words page. I see Lieven has added information on all of them. Thanks for updating the info.

**Pull request merged**